### ISSUE TITLE
Python version 3.7.3 is not supported by the project (^3.9)

### ISSUE THREAD

**Issue by aturX** [janezpodhostnik/flow-py-sdk#1]
Can I only use Python 3.9 mode now?

![image](https://user-images.githubusercontent.com/34887700/110217919-c6a74080-7ef1-11eb-9865-cf918dd765f2.png)

**Comment by aturX** [comment]
> Can I only use Python 3.9 mode now?
> 
> ![image](https://user-images.githubusercontent.com/34887700/110217919-c6a74080-7ef1-11eb-9865-cf918dd765f2.png)

3.7 Will it be more stable?

**Comment by aturX** [comment]
I am sorry to say that I have tried and failed to use `poetry run examples` in windows10.  
My environment is working correctly,  I can launch  `python run_examples.py` directly, but not through Poetry in Window 10. 

I failed to use `poetry install`

![image](https://user-images.githubusercontent.com/34887700/110220489-00337800-7f01-11eb-8bde-a001fbe2add8.png)

 
Is this a poetry question? I'm not sure 

https://github.com/python-poetry/poetry/issues/2629
